.github/workflows/upload-and-deploy.yaml

```diff
@@ -97,6 +97,19 @@ jobs:
           aws_secret_access_key: ${{ secrets.CI_USER_SECRET_ACCESS_KEY }}
           role_to_assume: ${{ secrets.CI_ROLE_TO_ASSUME }}
 
+      - name: Login to Amazon ECR
```

Comment:
This builds and pushes every time we merge a PR.

Reply:
Thanks - thats what I meant by push - should have been more clear.  I am ok with this method even if this gets triggered often.  (My only concern is if an update causes the job to fail ..)

Down the line we might still have to think about a way to auto rebuild images to meet security standards.

Reply:
Your idea of having a github action which runs on a cron schedule that builds and pushes this image regularly is a good one, but since we only need this image for the next few months and we will be constantly merging PRs, this method should be sufficient.

Reply:
Thanks - feel free to merge.